frame/staking/src/lib.rs

```diff
@@ -1059,28 +1074,46 @@ decl_storage! {
 		/// The earliest era for which we have a pending, unapplied slash.
 		EarliestUnappliedSlash: Option<EraIndex>;
 
+		/// The last planned session scheduled by the session pallet.
+		///
+		/// This is basically in sync with the call to [`SessionManager::new_session`].
+		/// TODO: TWO_PHASE: needs care to set the initial value upon migration.
+		pub CurrentPlannedSession get(fn current_planned_session): SessionIndex;
+
 		/// Snapshot of validators at the beginning of the current election window. This should only
 		/// have a value when [`EraElectionStatus`] == `ElectionStatus::Open(_)`.
+		///
+		/// DEPRECATED.
```

Comment:
If we go for switching latter I would say it is deprecated, but just have comment to mark it deprecated once switched.

Because these are the actual election informations

Reply:
Will change to `TODO: TWO_PHASE: ...` this is the comment patter that I am using for important things until we switch.